### ISSUE TITLE
Mouaaaahhhhhaaahhhhaahh

### ISSUE THREAD

**Issue by ontologiae** [Herzult/SimplePHPEasyPlus#3]
Ohhhohhhh
:-D
Lol

**Comment by Herzult** [comment]
Interesting, I'll keep it in mind for next releases.

**Comment by dbonsch** [comment]
Nice, that would be really a decent feature.

**Comment by thers** [comment]
You made my day :D But it's still a kind of time wasting, i think :D

**Comment by yuters** [comment]
:+1:

**Comment by vamsiikrishna** [comment]
good way to learn/teach oo php I guess

**Comment by iamwavecut** [comment]
Definetely needs one additional abstraction level to manage dependencies, service provider or use of dependency injection, its quite popular now!

**Comment by lavoiesl** [comment]
Yeah, and some classes are pretty simple, you should a couple of Abstract classes to factorize the whole thing

**Comment by jbeja** [comment]
Hahhahahahhahahahahahahahhaahhahahahahahhahahahahahahahhahahahahahahahah

**Comment by DjebbZ** [comment]
Best proof that PHP is becoming Java

**Comment by resurtm** [comment]
> Ohhhohhhh

I disagree. `Ohhhoohhhh` seems much better.

**Comment by felds** [comment]
:eggplant:

**Comment by c9s** [comment]
Ahahhahahahhahahahahhaa
